### PR TITLE
Improve TOC heading spacing

### LINKED ISSUE #495: Better multiline headings in TOC
In a page's table of contents, multi-line headings are difficult to discern. Instead, they should be grouped together.

image

### PR DESCRIPTION
**Pull request by dropforge**
Remove spacing between lines of the same multi-line heading and add spacing to the next item in the table of contents. Closes #495

image

### CODE REVIEW

File: myhpi/static/js/myHPI.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ const localizeLastPublished = () => {
         let abbr = document.createElement("abbr");
         abbr.setAttribute("title", lastPublishedLocalized)
         lastPublished.removeAttribute("title")
+        lastPublished.innerHTML = lastPublished.textContent.trim()
```
Comment (jeriox) [inline myhpi/static/js/myHPI.js:73]: why is this change necessary?

**Review by jeriox — APPROVED** [review]

File: myhpi/static/js/myHPI.js (hunk 1 above)
Comment (frcroth) [reply myhpi/static/js/myHPI.js:73]: Does this also work with innerText to get rid of this security warning?
